djstripe/models.py

```diff
@@ -287,6 +251,12 @@ def process_transfer(cls, event, transfer):
         if event.kind == "transfer.updated":
             obj.update_status()
 
+    @staticmethod
+    @webhook.handler(["transfer", ])
+    def webhook_handler(event, data, cat, sub_cat):
```

Comment:
If the handler call expects a list, you can leave out the trailing comma.

Also, are ``cat``, ``sub_cat`` the same as ``event_type`` and ``event_subtype`` from the ``Event`` model? If so, I thing it would be best to stick with one for consistency. I think ``event_category`` and ``event_subcategory`` would work. What do you think?